dev/tools/controllerbuilder/cmd/codebot/main.go

```diff
@@ -52,6 +52,8 @@ type Options struct {
 	UIType   string
 	Project  string
 	Location string
+
+	Model string
```

Comment:
Is there a default model for codebot chat?

/cc @barney-s this PR may affect how conductor calls codebot.

Reply:
Good call.  I also need to fix how project & location are plumbed through to vertex